tasks/main.yml

```
@@ -31,28 +31,17 @@
   when: tomcat_conf_found.stat.exists == false
   tags: ['tomcat']
 
-- name: secure catalina.policy
+- name: secure catalina base confg files 
   file:
-    path: '{{ tomcat_catalina_base }}/conf/catalina.policy'
-    owner: '{{ tomcat_user }}'
-    group: '{{ tomcat_group }}'
-    mode: 0640
-  tags: ['tomcat']
-
-- name: secure catalina.properties
-  file:
-    path: '{{ tomcat_catalina_base }}/conf/catalina.properties'
-    owner: '{{ tomcat_user }}'
-    group: '{{ tomcat_group }}'
-    mode: 0640
-  tags: ['tomcat']
-
-- name: secure web.xml
-  file:
-    path: '{{ tomcat_catalina_base }}/conf/web.xml'
+    path: '{{ tomcat_catalina_base }}/conf/{{ item }}'
     owner: '{{ tomcat_user }}'
     group: '{{ tomcat_group }}'
     mode: 0640
```

While it won't bite you here because of the leading `0`, you should quote the mode in file modules to prevent it accidentally being interpreted as a decimal rather than an octal.

```suggestion
    mode: '0640'
```